ledger/participant-integration-api/src/test/lib/scala/platform/store/backend/StorageBackendTestsIngestion.scala

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.platform.store.backend
+
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+private[backend] trait StorageBackendTestsIngestion extends Matchers with StorageBackendSpec {
+  this: AsyncFlatSpec =>
+
+  behavior of "StorageBackend (ingestion)"
+
+  import StorageBackendTestValues._
+
+  it should "ingest a single configuration update" in {
+    val someOffset = offset(1)
+    val dtos = dtoConfiguration(someOffset).toVector
+    for {
+      _ <- executeSql(backend.initializeParameters(someIdentityParams))
+      _ <- executeSql(ingest(dtos, _))
+      configBeforeLedgerEndUpdate <- executeSql(backend.ledgerConfiguration)
+      _ <- executeSql(backend.updateLedgerEnd(ParameterStorageBackend.LedgerEnd(someOffset, 0)))
+      configAfterLedgerEndUpdate <- executeSql(backend.ledgerConfiguration)
+    } yield {
+      // The first query is executed before the ledger end is updated.
+      // It should not see the already ingested configuration change.
+      configBeforeLedgerEndUpdate shouldBe empty
+
+      // The second query should now see the configuration change.
+      configAfterLedgerEndUpdate should not be empty
```

Comment:
Good point. In general, the StorageBackend suite does not verify the serialization and deserialization of values like Daml-Lf values or Daml-Lf packages. They are all treated as opaque blobs.
`StorageBackend.ledgerConfiguration` is special because it returns a deserialized value, whereas all other read methods return streams or arrays of bytes. I have changed this line to assert on the configuration contents.